main.go

```diff
@@ -26,11 +31,17 @@ func main() {
 	}
 	defer os.Remove(file.Name())
 
-	err = s.Exec(
+	args := []string{
 		"-silent", "-no-fallback", "-pipeline", "-tech-detect",
 		"-json", "-o", file.Name(),
 		"-l", options.Input,
-	)
+	}
+
+	if *userAgent != "" {
```

Comment:
Same as above, as in, `*userAgent` will always be something so not sure if this `if` condition makes sense.

Reply:
As the help says, "set to empty to use random agent"

Reply:
`--ua ""`? Thought this was ignored and the default was used instead 👍